README.md

```diff
@@ -2,15 +2,15 @@
 
 ## Media Assets
 
-This project utilizes media assets that are created by [夜夢（よるむ], who gave permission to modify and redistribute their work.
+This project utilizes media assets that are created by [夜夢（よるむ)](https://www.pixiv.net/en/users/345405), who gave permission to modify and redistribute their work.
 
 Please note that if you plan to use or redistribute these assets, you must adhere to the terms of the license. See the [LICENSE](COPYING.CC-BY-NC-SA.4.0.md) file for details.
 
-### Credits
+## Credits
 
-- Original Media Assets: [夜夢（よるむ）]
+- Media Assets by: 夜夢（よるむ)
 - [Author's Pixiv Profile](https://www.pixiv.net/en/users/345405)
-- Cursors were ported by: [muha0644](https://www.pling.com/u/muha0644)
+- Some cursors were ported by [muha0644](https://www.pling.com/u/muha0644), which were used as reference.
```

Comment:
reference as in, i looked at their ports and mapped the windows names to x11 with one of them